tests/aws/services/cloudwatch/test_cloudwatch.py

```diff
@@ -148,10 +148,14 @@ def _get_metric_data_sum():
             assert 2 == len(response["MetricDataResults"])
 
             for data_metric in response["MetricDataResults"]:
+                # TODO: there's an issue in the implementation of the service here.
+                #  The returned timestamps should have the seconds set to 0
```

Comment:
Just something I saw while debugging this. This isn't really related to the issue here though which could still happen even if this was fixed.